package.json

```diff
@@ -5,7 +5,7 @@
 	"version": "0.2.1",
 	"publisher": "MicroProfile-Community",
 	"preview": true,
-	"license": "EPL-2.0",
+	"license": "Apache 2.0",
```

Comment:
The license id should be `Apache-2.0` according to https://docs.npmjs.com/files/package.json#license and https://spdx.org/licenses/